mm/mm_heap/mm_free.c

```diff
@@ -90,9 +90,9 @@ void mm_free(FAR struct mm_heap_s *heap, FAR void *mem)
     {
       kasan_unpoison(mem, mm_malloc_size(mem));
 
-      /* We are in IDLE task & can't get sem, or meet -ESRCH return,
-       * which means we are in situations during context switching(See
-       * mm_takesemaphore() & getpid()). Then add to the delay list.
+      /* Meet -ESRCH return, which means we are in situations
+       * during context switching(See mm_takesemaphore() & getpid()).
+       * Then add to the delay list.
        */
```

Comment:
@GUIDINGLI 

Do these comment changes relate to the bug fix in mm_sem?

Reply:
Yes, after this patch, idle will not return false(unless fatal error happened), so need update the comment